### ISSUE TITLE
[BUG] Installation fails

### ISSUE THREAD

**Issue by ecod3r** [MilesCranmer/PySR#118]
**Describe the bug**
After installing from conda in a Python 3.9 environment, installation fails with the following

`julia.core.JuliaError: Exception 'ReadOnlyMemoryError()' occurred while calling julia code Pkg.add(spec, io=stderr)`

**Version (please complete the following information):**
- OS: Ubuntu
- Version 20.04.2 LTS (Focal Fossa)
- Does the bug appear with the latest version of PySR? Yes

**Configuration**
- What are your PySR settings? None. The installed version is 0.7.9 (pyh6c4a22f_0 from conda-forge)
- What dataset are you running on? None

**Error message**
```
To silence this warning, you can run pysr.silence_julia_warning() after importing pysr.
  warnings.warn(
  Activating new environment at `~/.julia/environments/pysr-0.7.9/Project.toml`
     Cloning git-repo `https://github.com/MilesCranmer/SymbolicRegression.jl`
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home//anaconda3/envs/stable/lib/python3.9/site-packages/pysr/sr.py", line 46, in install
    _add_sr_to_julia_project(Main, io_arg)
  File "/home//anaconda3/envs/stable/lib/python3.9/site-packages/pysr/sr.py", line 343, in _add_sr_to_julia_project
    Main.eval(f"Pkg.add(spec, {io_arg})")
  File "/home//anaconda3/envs/stable/lib/python3.9/site-packages/julia/core.py", line 621, in eval
    ans = self._call(src)
  File "/home/em/anaconda3/envs/stable/lib/python3.9/site-packages/julia/core.py", line 549, in _call
    self.check_exception(src)
  File "/home/em/anaconda3/envs/stable/lib/python3.9/site-packages/julia/core.py", line 603, in check_exception
    raise JuliaError(u'Exception \'{}\' occurred while calling julia code:\n{}'
julia.core.JuliaError: Exception 'ReadOnlyMemoryError()' occurred while calling julia code:
Pkg.add(spec, io=stderr)
```

**Additional context**

**Comment by MilesCranmer** [comment]
Where is your anaconda3 folder located? The error trace back seems to reference two different folders: `/home/anaconda3` and `/home/em/anaconda3` - maybe do you have two conflicting installations?

**Comment by MilesCranmer** [comment]
Very strange. Any chance you installed conda with sudo? It looks like it’s unable to write because of permission issues.

**Comment by MilesCranmer** [comment]
This looks similar in some ways: https://github.com/JuliaLang/julia/issues/39396

Maybe you are running out of RAM? How much RAM do you have?

**Comment by ecod3r** [comment]
1 TB, half of which is available - can free up more, though I'd be *very* surprised if that mattered. Info, as printed by `pysr.install()`

```
Platform Info:
  OS: Linux (x86_64-conda-linux-gnu)
      Ubuntu 20.04.2 LTS
  uname: Linux 5.4.0-77-generic #86-Ubuntu SMP Thu Jun 17 02:35:03 UTC 2021 x86_64 x86_64
  CPU: AMD EPYC 7702P 64-Core Processor: 
                  speed         user         nice          sys         idle          irq
       #1-128  1499 MHz  1189067606 s      89796 s   73479266 s  8341110380 s          0 s
       
  Memory: 1003.5033874511719 GB (455850.2890625 MB free)
```

**Comment by MilesCranmer** [comment]
Also, make sure your conda is up to date (`conda update -n base conda`), but I'm not sure if that's relevant for this.

**Comment by MilesCranmer** [comment]
Presumably fixed by #535
